tests/Unit/Countries/US/Kentucky/V20190101/local/CarrollCountyTest.php

```diff
@@ -109,12 +109,28 @@ public function provideTestData(): array
                     ->setExpectedAmountInCents(100)
                     ->build()
             ],
-            'over max tax' => [
+            'close to max tax, take difference not full amount' => [
+                $builder
+                    ->setWagesInCents(10000)
+                    ->setYtdWagesInCents(0)
+                    ->setYtdLiabilitiesInCents(4999950)
+                    ->setExpectedAmountInCents(50)
+                    ->build()
+            ],
+            'at max tax' => [
                 $builder
                     ->setWagesInCents(10000)
                     ->setYtdWagesInCents(0)
                     ->setYtdLiabilitiesInCents(5000000)
-                    ->setExpectedAmountInCents(100)
+                    ->setExpectedAmountInCents(0)
+                    ->build()
+            ],
+            'over max tax' => [
+                $builder
+                    ->setWagesInCents(10000)
+                    ->setYtdWagesInCents(0)
+                    ->setYtdLiabilitiesInCents(5000100)
```

Comment:
this shouldn't be right should it? 5m cents for $50k shoudn't hit the max should it?

Reply:
Max is already hit there. The YtdLiabilities is what has been paid already and it is over max.

Reply:
ah, right, that's amount paid not wages earned :duh:

Reply:
It is confusing which goes to which, lots of dumps to figure out the values